pkgs/development/python-modules/home-assistant-chip-core/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchurl
+, pythonOlder
+
+# build
+, autoPatchelfHook
+
+# runtime
+, openssl_1_1
+
+# propagates
+, coloredlogs
+, construct
+, dacite
+, rich
+, pyyaml
+, ipdb
+, deprecation
+, mobly
+, pygobject3
+}:
+
+let
+  wheels = {
+    "aarch64-linux" = fetchurl {
+      url = "https://files.pythonhosted.org/packages/38/04/a5886ac7356d1202ec6586bf01e24201fefc52d10f54795229558b81f039/home_assistant_chip_core-2022.12.0-cp37-abi3-manylinux_2_31_aarch64.whl";
+      hash = "sha256-oNqrvbzXeXpMG3v9RK6kppONH4n7xLVaJCEFXxVj2jE=";
+    };
+    "x86_64-linux" = fetchurl {
+      url = "https://files.pythonhosted.org/packages/8b/3a/5c68a504407188c3b6b183fc2958b4fe5aafe17ebc0c442f1e3587c090c7/home_assistant_chip_core-2022.12.0-cp37-abi3-manylinux_2_31_x86_64.whl";
+      hash = "sha256-S5n1MUig8ZDSLgWeVmu+5qLZ4kfHQUC9qZcVfM8rPvw=";
+    };
```

Comment:
just that it is not forgotten in the chat history

https://files.pythonhosted.org/packages/cp37/h/home_assistant_chip_core/home_assistant_chip_core-2022.12.0-cp37-abi3-manylinux_2_31_x86_64.whl

Reply:
Wondering if this is an improvement at all.

[`140fd27` (#210297)](https://github.com/NixOS/nixpkgs/pull/210297/commits/140fd2715d1787af3db0a8cb5d76950ff0c7fb0d)